include/magma/vk/Window.h

```diff
@@ -21,21 +21,21 @@ class Window {
     VkExtent2D getResolution() const { return {_width, _height}; }
     void       updateResolution();
 
-    GLFWwindow * getGLFWp()         { return _window;         }
-    VkSurfaceKHR getSurface() const { return _surface;        }
-    bool         wasResized() const { return _wasResized;     }
-    Keyboard *   getKeyboard()      { return _keyboard.get(); }
-    Mouse *      getMouse()         { return _mouse.get();    }
+    GLFWwindow *   getGLFWp()         { return _window;         }
```

Comment:
What does 'p' in 'GLFW**p**' stands for?

Reply:
'p' means "pointer" :D

Reply:
renamed this method to getGlfwWindow()